src/renderers/WebGLRenderer.js

```diff
@@ -856,7 +856,16 @@ class WebGLRenderer {
 
 			if ( object.isBatchedMesh ) {
 
-				renderer.renderMultiDraw( object._multiDrawStarts, object._multiDrawCounts, object._multiDrawCount );
+				// TODO: implement this field in BatchedMesh or InstancedBatchedMesh
+				if ( object._multiDrawInstances !== undefined ) {
```

Comment:
You are planning to make this public somehow so app code is not required to define a private field, right?

How about you expose it directly as `BatchedMesh.multiDrawInstances` with `null` as the default?

Reply:
I kept this field private to align with the other multi-draw properties. Defaulting to `null` is good, but what about either:

Aliasing to a public property not mentioning multi-draw:

```js
get instanceCounts() {

	return this._multiDrawInstances;

}
set instanceCounts( instanceCounts ) {

	this._multiDrawInstances = instanceCounts;

}

```

and/or implementing `setInstancesAt` in `BatchedMesh` (or later `InstancedBatchedMesh`)?

```js
getInstancesAt( id ) {

	if ( this._multiDrawInstances === null ) return null;

	return this._multiDrawInstances[ id ];

}

setInstancesAt( id, instanceCount ) {

	if ( this._multiDrawInstances === null ) {

		this._multiDrawInstances = new Int32Array( this._maxGeometryCount ).fill( 1 );

	}

	this._multiDrawInstances[ id ] = instanceCount;

	return id;

}
```

Reply:
`setInstancesAt()` looks easier to use since the user would not have to deal with `Int32Array`.